### DISCUSSION TITLE
Akash Analytics Grant Proposal

### DISCUSSION THREAD

**Discussion by baktun14** [akash-network/community#220] [Governance Proposals]
## Summary

As the Akash Network continues to grow, there is a critical need for robust analytics to track network activity and ensure its health. We propose to lead the charge in developing and providing these analytics, building on our 2+ years of experience providing this service for the network. Our goal is to provide access to all the stats about the network activity and health mentioned in the sig-analytics prd and offer real-time monitoring, visualization, and analysis of the network's performance, security, and scalability.

## Proposed Solutions

To achieve this goal, we will implement and provide access to all the stats about the network activity and health mentioned in the sig-analytics prd, including the addition of GPU stats once it's on the mainnet. We will offer access to this data through our custom dashboard on Cloudmos (https://deploy.cloudmos.io/analytics) and through our new BigQuery integration that we're currently working on. This will allow Akash Network developers and stakeholders to access the data they need to make informed decisions about the network's direction.

In addition to providing access to the data, we will also improve our dashboards and charts with new stats and features to enhance the user experience. We will also provide continuous support and maintenance for the web3index.org integration to ensure the analytics remain up-to-date and relevant.

Link to the prd: https://github.com/akash-network/community/blob/main/sig-analytics/prd.md

## Funding Request & Timeline

Working with such a large dataset is costly and challenging, especially since we support data from the genesis block. To successfully implement the sig-analytics prd and provide access to the data for the next 12 months, we request 45k USD in AKT.

**Timeline**

1. BigQuery integration - May 31, 2023
2. Addition of GPU stats and implementation of sig-analytics prd - Jun 30, 2023
3. Improvement of dashboards and charts with new stats and features - Jul 31, 2023

Once the first 12 months are completed, a yearly maintenance fee will be inquired of 5k USD paid upfront. Maintenance includes chain upgrades, server costs and man power.

### Conclusion

Providing access to robust analytics is essential for the growth and success of the Akash Network. Our team has a wealth of experience in developing and providing such services, and we are confident that we can deliver on our proposed solutions in a timely and effective manner. With your support and cooperation, we can help take the Akash Network to the next level.

_Reference_

https://deploy.cloudmos.io/analytics
https://deploy.cloudmos.io/providers

**Comment by gosuri** [comment]
Great proposal and super valuable for Akash. When will the code be open-sourced?

**Reply by baktun14** [reply]
Hey Greg, thanks for considering our grant proposal.

However, the ask to open-source our indexer code is a tough one for us. We've spent considerable time and resources crafting unique algorithms, which are a significant part of our competitive advantage. While we respect the ethos of transparency in the blockchain and open-source community, we need to balance that with protecting our proprietary technology. Not to forget, our indexer is made to work with any Cosmos chain.

That's why I suggested we give the community full access to our data through the BigQuery integration. This approach was agreed upon during the sig-analytics meetings as a fitting compromise. Our intention here is to ensure we can maintain the balance of transparency and protection of our proprietary tech while continuing to contribute valuable services to the Akash Network.

**Reply by ghost** [reply]
Explanation for thumbs down: I'm pro open-source guy. I don't like anything that's a black box. Hence thumbs down.

**Reply by anilmurty** [reply]
hey @baktun14 - thanks for the proposal. I think we've been pretty clear about our open source stance (and impose the same requirement on ourselves) in all sig meetings.
